public/app/features/datasources/components/picker/DataSourceModal.tsx

```diff
@@ -212,21 +214,31 @@ function getDataSourceModalStyles(theme: GrafanaTheme2) {
 
       ${theme.breakpoints.down('md')} {
         width: 100%;
-        height: 53%;
         padding-left: 0;
-        padding-top: ${theme.spacing(4)};
+        flex: 0;
```

Comment:
and we would need this to make it possible
```suggestion
        flex: 1;
```